js/dmp.playlist.js

```diff
@@ -96,7 +96,7 @@ dmp.playlist.loadPlaylist = function(file) {
  */
 dmp.playlist.handleErrors = function(e) {
   if(e.type == gapi.drive.realtime.ErrorType.TOKEN_REFRESH_REQUIRED) {
-    dmp.auth.autoRefreshAuth();
+    console.log("You have been signed out from your Google Account.");
```

Comment:
It is actually a bug with the realtime API client library (a developer just raised it on our issues tracker too - https://github.com/google/google-api-javascript-client/issues/287#issuecomment-307666854). I will have to fix the library internally to make this work.